api/v1beta2/condition_types.go

```diff
@@ -19,15 +19,23 @@ package v1beta2
 const SourceFinalizer = "finalizers.fluxcd.io"
 
 const (
+	// ArtifactInStorageCondition indicates the availability of Artifact of the
+	// Source in the storage.
```

Comment:
For improved clarity:
```suggestion
	// ArtifactInStorageCondition indicates the availability of the Source's Artifact
	// in the storage.
```

Or remove the reference to source altogether:
```suggestion
	// ArtifactInStorageCondition indicates the availability of the Artifact in the storage.
```

Reply:
Added the latter.